### PR TITLE
Propose shrutir25 for Release Shadow and anhowe for Manual Upgrade

### PR DESCRIPTION
**Pull request by shrutir25**

### CODE REVIEW

**Comment by shrutir25** [comment]
Have signed CLA

**Comment by shrutir25** [comment]
Have signed CLA. Check again

**Comment by calebamiles** [comment]
@shrutir25, I think we've decided that it's probably ok to combine the Automatic and manual upgrade lead into a single role now that most of the manual upgrade tests have been automated. Would you mind combining both roles into "Upgrade Testing" with @anhowe proposed as for the role? Thanks!

File: release-1.7/release_team.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
+|  Lead | Dawn Chen (dchen1107) | Shruti Ramesh (shrutir25) |
+|  Secondary |  | |
 |  Features |Ihor Dvoretskyi (idvoretskyi) | |
 |  Release branch |Wojciech Tyczynski (wojtek-t)| |
 |  Infra | | | |
 |  Docs |Andrew Chen (chenopis) | |
 |  Bugs | | | |
 |  Testing | | | |
-|  Manual upgrade | | | |
+|  Manual upgrade | Anthony Howe (anhowe) | | |
```
Comment (calebamiles) [inline release-1.7/release_team.md]: manual + automatic upgrade can probably be combined at this point into one role served by Anthony

**Comment by idvoretskyi** [comment]
What about the @marun's role? Has he agreed to step back or voting is needed?

**Comment by marun** [comment]
@idvoretskyi:  This is the first I'm hearing of it.  @shrutir25 may not have seen the thread where I deferred to @dchen1107 and offered to shadow her.

@dchen1107: @calebamiles suggested that both @shrutir25 and I shadow you.  Would that make sense, or does one of us need to be chosen?

File: release-1.7/release_team.md [hunk 2]
```diff
@@ -1,12 +1,11 @@
 |  **Role** | **Name** (**GitHub/Slack ID**)  | **Shadow Name(s) (GitHub/Slack ID), ...**
 |  ------ | ------ | ------ |
-|  Lead | Dawn Chen (dchen1107) | Maru Newby (marun) |
+|  Lead | Dawn Chen (dchen1107) | Shruti Ramesh (shrutir25) |
```
Comment (idvoretskyi) [inline release-1.7/release_team.md]: Shadow lead question between Shruti and Maru nominations has to be resolved.

**Comment by calebamiles** [comment]
Let's keep all the conversation in the [Google Group](https://groups.google.com/forum/?utm_medium=email&utm_source=footer#!msg/kubernetes-dev/M-YDooHvE3s/nVUGs1YSBQAJ)

**Comment by pwittrock** [comment]
Upgrade is jeff regan

**Comment by idvoretskyi** [comment]
Closing as the release team is ready https://github.com/kubernetes/features/pull/295

**Pull request closed without merging**